crates/polars-ops/src/series/ops/unique.rs

```diff
@@ -25,6 +26,150 @@ where
     out.into_inner()
 }
 
+fn unique_counts_boolean_helper(ca: &BooleanChunked) -> IdxCa {
+    if ca.is_empty() {
+        return IdxCa::new(ca.name(), [] as [IdxSize; 0]);
+    }
+
+    let ca = ca.rechunk();
```

Comment:
@ritchie can answer better here, but I think if we're spending the energy rechunking a series (which mem copies), the original array should receive the benefit as well, which you can do by making `ca` `mut` and calling `*ca.rechunk()`.

Reply:
Might be. However, I can't see this being done anywhere else in the codebase, which makes me question whether it should be done, so will wait for Ritchie's answer.

I guess you meant @ritchie46 btw.